system/lib/libc/emscripten_time.c

```diff
@@ -25,6 +25,13 @@ void _localtime_js(const time_t *restrict t, struct tm *restrict tm);
 void _gmtime_js(const time_t *restrict t, struct tm *restrict tm);
 double _emscripten_date_now();
 double emscripten_get_now_res();
+void _emscripten_get_timezone_js(char* buffer, int length);
+
+const char* emscripten_get_timezone() {
+  static thread_local char buffer[128];
```

Comment:
Where does 128 come from?  Might be worth a comment.

Reply:
good call, looked deeper into the Tz database to properly size, also added a comment